app/presenters/question_presenter.rb

```diff
@@ -71,6 +71,14 @@ def response_label(value)
     value
   end
 
+  def change_link_track_label(value)
+    if redacted?
```

Comment:
technically would be slightly more performant if these were swapped but it wouldn't read as nicely so I don't really mind

Reply:
although actually no it wouldn't because it would have to check `redacted?` regardless so ignore me entirely!